src/node_errors.cc

```diff
@@ -454,6 +454,38 @@ void OnFatalError(const char* location, const char* message) {
   ABORT();
 }
 
+v8::ModifyCodeGenerationFromStringsResult ModifyCodeGenerationFromStrings(
```

Comment:
Is this essentially just a hook that v8 exposes that happens after `eval` is called, before the value is returned?

Reply:
This is the hook that will be called before v8 tries to compile the dynamic sources like `eval` and `new Function`.